frontend/components/organiser/event/forms/AddFormResponseDialog.tsx

```
@@ -0,0 +1,142 @@
+              leaveTo="opacity-0 scale-95"
+            >
+              <DialogPanel className="w-full max-w-4xl transform overflow-hidden rounded-2xl bg-white text-left align-middle shadow-xl transition-all flex flex-col max-h-[90vh]">
+                {/* Header */}
+                <div className="flex justify-between items-center p-6 border-b border-gray-200 shrink-0">
+                  <DialogTitle as="h3" className="text-xl font-bold text-gray-900">
+                    Add Manual Form Response
+                  </DialogTitle>
+                  <button
+                    onClick={onClose}
+                    className="p-1 hover:bg-gray-100 rounded-full transition-colors focus:outline-none"
+                  >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)